packages/terra-site/webpack.config.js

```diff
@@ -11,18 +11,13 @@ const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
 
 module.exports = {
   entry: {
-    'babel-polyfill': 'babel-polyfill',
```

Comment:
babel-transform-runtime does the same thing as babel-polyfill but scoped (rather than global). I need to verify it works as expected and if not I'll revert babel-polyfill.